api/api/services/internal_tasks/meta_agent_service_test.py

```diff
@@ -804,3 +804,122 @@ async def test_fetch_meta_agent_context(
 
             # For reviewed_input_count
             assert result.reviewed_input_count == expected_context.reviewed_input_count
+
+    @pytest.mark.asyncio
```

Comment:
not needed

Reply:
right deleted https://github.com/WorkflowAI/WorkflowAI/pull/91/commits/9ceef69a03d9b7a6ef529b80eb25af8cd6324485